homeassistant/components/overkiz/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
```

Comment:
```py
    OverkizSwitchDescription(
        key=UIWidget.RTD_INDOOR_SIREN,
        turn_on=OverkizCommand.ON,
        turn_off=OverkizCommand.OFF,
        icon="mdi:bell",
    ),
    OverkizSwitchDescription(
        key=UIWidget.RTD_OUTDOOR_SIREN,
        turn_on=OverkizCommand.ON,
        turn_off=OverkizCommand.OFF,
        icon="mdi:bell",
    ),
```
Arent these sirens?

Reply:
Yes, but they are stateless and thus couldn't be mapped to the [Siren entity](https://developers.home-assistant.io/docs/core/entity/siren/) if I understand correctly.